projects/angular/src/forms/combobox/_combobox.clarity.scss

```
@@ -323,36 +160,21 @@
 
     &:not(.disabled):hover,
     &:focus-within {
-      @include mixins.css-var(
-        background-color,
-        clr-combobox-hover-background-color,
-        combobox-variables.$clr-combobox-input-background-color,
-        variables.$clr-use-custom-properties
-      );
+      background-color: combobox-variables.$clr-combobox-hover-background-color;
     }
   }
 
   .clr-combobox-trigger {
     &:disabled {
-      color: var(--clr-btn-link-disabled-color, #666);
-      opacity: 0.4;
+      color: tokens.$cds-alias-object-interaction-color-disabled;
+      cds-icon[shape='angle'],
+      clr-icon[shape='angle'] {
+        color: tokens.$cds-alias-object-interaction-color-disabled;
+      }
     }
-
     //Dimensions
-    @include mixins.css-var(
-      padding-left,
-      clr-combobox-trigger-padding-left,
-      combobox-variables.$clr-combobox-trigger-left-padding,
-      variables.$clr-use-custom-properties
-    );
-
-    @include mixins.css-var(
-      padding-right,
-      clr-combobox-trigger-padding-right,
-      // not an error; we're reusing "left" variable for the "right" fallback
-      combobox-variables.$clr-combobox-trigger-left-padding,
-      variables.$clr-use-custom-properties
-    );
+    padding-left: combobox-variables.$clr-combobox-trigger-padding-left;
+    padding-right: combobox-variables.$clr-combobox-trigger-padding-right;
```

We should keep the naming convention or use a deprecation clause in _variables.combobox.scss.

```suggestion
    padding-left: combobox-variables.$clr-combobox-trigger-left-padding;
    padding-right: combobox-variables.$clr-combobox-trigger-padding-right;
```

---

Reverted.